### DISCUSSION TITLE
docker container options with platform directive?

### DISCUSSION THREAD

**Discussion by smaring** [nektos/act#1433] [Q&A] [answered]
We use self-hosted runners within our organization to interact with on-prem Artifactory and Sonar instances.  In order to access these instances we must connect to the corporate network via VPN.  My desktop/workstation is native Ubuntu.  By default, the Docker networking is "bridged".  I am unable to get network traffic to go out the VPN unless I start Docker containers with "host" networking.  There doesn't seem to be a way to change the default networking so that you don't have to override it when starting a container.  With nektos/act I can specify the image I want it to use, but is there a way to specify any further options like networking?

```
act push \
	--workflows .github/workflows/merge-to-master.yml \
	--secret     MY_SECRET=****** \
	--platform   self-hosted=my-docker-image
```

**Comment by KnisterPeter** [comment, marked as answer]
I think you can use latest `master` version with this feature: https://github.com/nektos/act/pull/1462
It's not yet documented on the README.
